src/pages/projects.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react"
+import type { HeadFC, PageProps } from "gatsby"
+import { graphql } from 'gatsby'
+import ProjectCard from '../components/projectcard';
+import '../styles/style.css'
+import { getImage } from "gatsby-plugin-image";
+
+const pageStyles = {
+  color: "#232129",
+  padding: 0,
+  fontFamily: "-apple-system, Roboto, sans-serif, serif",
+}
+
+import comp_image from '../images/image.png';
+
+// // GraphQL query to get the image data
+// export const query = graphql`

```

Comment:
did image loading code work - if so, please use it - gatsby then does some image optimising magic, causing page to load faster - if doesn't work - ignore for now